test/unit/services/service.service.test.js

```diff
@@ -213,13 +210,12 @@ describe('service service', function () {
           './clients/adminusers.client': adminusersClientStub
         })
 
-      serviceService.updateServiceName(externalServiceId, newServiceName, correlationId).should.be.fulfilled
-        .then((service) => {
-          setTimeout(() => {
-            expect(patchServiceName.callCount).to.equal(2)
-          }, 250)
-          expect(JSON.stringify(service)).to.deep.equal('{"gatewayAccountIds":[10,9,"DIRECT_DEBIT:adashdkjlq3434lk"]}')
-        }).should.notify(done)
+      const service = await serviceService.updateServiceName(externalServiceId, newServiceName, correlationId)
+
+      setTimeout(() => {
+        expect(patchServiceName.callCount).to.equal(2)
+      }, 250)
```

Comment:
I don't think we need the timeout now we're awaiting the result?

Reply:
Done